client/my-sites/stats/stats-chart-tabs/chart-header.jsx

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
```

Comment:
We should probably start new component in TypeScript, where PropTypes wouldn't be necessary anymore.

Reply:
Yeah totally. Adding this to the followup PR which will also address your excellent suggestion above. I've been pondering how to use this as a slot so will try figure it out 😄

Reply:
issue created to followup here: https://github.com/Automattic/red-team/issues/254